### ISSUE TITLE
experiment/late fusion dataloader 

### ISSUE THREAD

**Issue by ChenKaiXuSan** [ChenKaiXuSan/Skeleton_ASD_PyTorch#10]
In late fusion, because dataset needs to return both dataloader (stance/swing).
The current method is to prepare two dataloaders separately, and then combine them into a dict to be passed into the train process.
The only issue is whether the order is maintained when selecting data from the dataloader.

The current dataloader, shuflle=false, batch size=1.
Avoids dropping the part that could cause problems, but whether it works for the whole data is undetermined.

**Comment by ChenKaiXuSan** [comment]
if shuflle=false, then can keep the sort with stance/swing.
But I think that would cause excess loading consumption.
The solution would be to rewrite the dataset load method.

**Comment by ChenKaiXuSan** [comment]
The dataset logic is now modified.
The stance/swing division is now done primarily from inside the dataset, rather than creating two dataloaders.
The purpose of this is to ensure that the order is consistent when loading, and to ensure the accuracy of the labels.
